src/components/select-project/select-project.component.ts

```diff
@@ -2,11 +2,14 @@ import {SelectProjectController} from './select-project.controller.ts';
 
 export const selectProject: angular.IComponentOptions = {
   bindings: {
-    selectedProject: '=',
+    selectedProject: '=?',
+    selectedProjectName: '@?',
```

Comment:
I suggest `preselectProjectName` so it's clear that the value is not updated on changes.